viewer/viewer-awt/src/main/java/org/icepdf/ri/common/views/PageViewComponentImpl.java

```diff
@@ -510,14 +510,14 @@ private void initializeAnnotationsComponent(Page page) {
         if (documentViewController.getAnnotationCallback() != null) {
             documentViewController.getAnnotationCallback().pageAnnotationsInitialized(page);
         }
-        if (annotations != null && annotations.size() > 0) {
+        if (annotations != null && !annotations.isEmpty()) {
             // we don't want to re-initialize the component as we'll
             // get duplicates if the page has be gc'd
             if (annotationComponents == null) {
                 annotationComponents = new ArrayList<>(annotations.size());
                 annotationToComponent = new HashMap<>(annotations.size());
                 Annotation annotation;
-                for (int i = 0, max = annotations.size(); i < max; i++) {
+                for (int i = 0; i < annotations.size(); i++) {
```

Comment:
We'll have to keep on eye on this change.  In the past I had scene some concurrence issues, but let's give it a try.

Reply:
Right, maybe it'd be better if I change it back then, but I'm not really seeing how a page could change it's annotations programmatically at this point.